src/apps/jsonld_converter/service/document/field.py

```diff
@@ -334,6 +335,60 @@ def _build_config(self, _cls: Type | None, **attrs):
 
         return config
 
+class ReproFieldParagraphText(ReproFieldBase):
+    INPUT_TYPE = "paragraphText"
+    RESPONSE_TYPE = ResponseType.PARAGRAPHTEXT
+
+    ld_correct_answer: str | None = None
+    ld_is_response_identifier: bool | None = None
+    ld_max_length: int | None = None
+    ld_value_type: str | None = None
+    ld_required_value: bool | None = None
+
+    is_multiple: bool = False
+    choices: list[dict] | None = None
+
+    @classmethod
+    def _get_supported_input_types(cls) -> list[str]:
+        return [cls.INPUT_TYPE]
+
+    async def _load_from_processed_doc(self, processed_doc: dict, base_url: str | None = None):
+        await super()._load_from_processed_doc(processed_doc, base_url)
+        self.ld_correct_answer = self.attr_processor.get_translation(processed_doc, "schema:correctAnswer", self.lang)
+
+    async def _process_ld_response_options(self, options_doc: dict, drop=False):
+        await super()._process_ld_response_options(options_doc, drop=drop)
+        self.is_multiple = self._get_ld_is_multiple(options_doc)
+        self.ld_is_response_identifier = self.attr_processor.get_attr_value(
+            options_doc, "reproschema:isResponseIdentifier"
+        )
+        self.ld_max_length = self.attr_processor.get_attr_value(options_doc, "reproschema:maxLength")
+        self.ld_required_value = self.attr_processor.get_attr_value(options_doc, "reproschema:requiredValue")
+        self.ld_value_type = self.attr_processor.get_attr_single(
+            options_doc, "reproschema:valueType", ld_key=LdKeyword.id
+        )
+
+        self.choices = self._get_ld_choices_formatted(options_doc)
```

Comment:
What does `choices` and `is_multiple` mean for this item type?

Such custom item types are not defined in JSON-LD ReproSchema, so I recommend ignoring the jsonld_converter for now, as it is currently unused and will require a significant effort to update / actualze.